wordpress-mobile/gutenberg-mobile
# Cover: Add Background Media button when only background colour is set

**Fixes** https://github.com/WordPress/gutenberg/issues/23877 (**parent** https://github.com/wordpress-mobile/gutenberg-mobile/issues/2275)

## Description
Please check `gutenberg` PR https://github.com/WordPress/gutenberg/pull/23878

To test:
* Add a cover block
* Press on a colour from the palette
* Verify that a new cover block with only colour is created
* Verify that the new icon is presented at the top-left of the component
* Tap on the icon and verify that the add media options are presented
* Add a media and verify that it is applied in the background

PR submission checklist:
